app/models.py

```diff
@@ -1,5 +1,7 @@
 from datetime import datetime
 from app import db
+from werkzeug.security import generate_password_hash, check_password_hash
+
 
 class User(db.Model):
     id = db.Column(db.Integer, primary_key=True)
```

Comment:
You can create a `BaseModel` class and have an id. (Every db table will have an id)
Your `Post` and `User` models (classes) can inherit `BaseModel` instead of db.Model.

So:
```python

class BaseModel(db.Model):
    id = db.Column(db.Integer, primary_key=True)

#Make your other models inherit base model
class User(BaseModel):
    # Remove the id

class Post(BaseModel):
    # Remove the id
```
This way you don't have to define an id each time you create a new model (post, user, comment etc)

Reply:
Still presists